client/blocks/edit-gravatar/style.scss

```diff
@@ -20,3 +20,31 @@
 		left: 50%;
 	transform: translate(-50%, -50%);
 }
+
+.edit-gravatar-modal.dialog.card {
```

Comment:
Do we need all 3 classes here to override the default specificity? Or can we get away with just `.edit-gravatar-modal`?